enhancements/two-nodes-openshift/2no.md

```diff
@@ -216,7 +239,12 @@ This will allow the use of a credential scoped to `ConfigMap`s in the `openshift
 
 The plan is for this to be changed by one of the nodes during pacemaker initialization. Pacemaker initialization should be initiated by CEO when it detects that the cluster controlPlane topology is set to `DualReplica`.
 
-#### Install Config with Fencing Credentials
+While set to `External`, CEO will still need to render the configuration for the etcd pod in a place where it can be consumed by pacemaker. This ensures that the pod
```

Comment:
we need the etcd **container** configuration, not the **pod**. Actually, the container we need isn't in the pod anymore after CEO disables etcd, IIRC

Reply:
I think I've resolved this. Please take look and let me know if the update from pod to container was sufficient to get the message across.